### PR TITLE
fix(curriculum): Improve punctuation/formatting in "Add Axes to a Visualization"

### PR DESCRIPTION
**Pull request by TyMick**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

1. Adds commas to [set off an appositive phrase](https://en.wikipedia.org/wiki/Comma#Parenthetical_phrases) ("<code>axisLeft()</code> and <code>axisBottom()</code>").
2. Adds appropriate [suspended hyphens](https://en.wikipedia.org/wiki/Hyphen#Suspended_hyphens) to "y- and x-axes".
3. Moves the parenthetical "&hairsp;'axes' is the plural form of 'axis'&hairsp;" to be part of the previous sentence. I think it fits better there, rather than as its own sentence.

### CODE REVIEW

**Review by erictleung — DISMISSED** [review]

File: curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/add-axes-to-a-visualization.english.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ forumTopicId: 301472
 ## Description
 <section id='description'>
 Another way to improve the scatter plot is to add an x-axis and a y-axis.
-D3 has two methods <code>axisLeft()</code> and <code>axisBottom()</code> to render the y and x axes, respectively. (Axes is the plural form of axis). Here's an example to create the x-axis based on the <code>xScale</code> in the previous challenges:
+D3 has two methods, <code>axisLeft()</code> and <code>axisBottom()</code>, to render the y- and x-axes, respectively ("axes" is the plural form of "axis"). Here's an example to create the x-axis based on the <code>xScale</code> in the previous challenges:
```
Comment (RandellDawson) [inline curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/add-axes-to-a-visualization.english.md:12]:
```suggestion
D3 has two methods, <code>axisLeft()</code> and <code>axisBottom()</code>, to render the y-axis and x-axis, respectively. Here's an example to create the x-axis based on the <code>xScale</code> in the previous challenges:
```

**Comment by tywmick** [comment]
Just cherry-picked a commit from another PR of mine, #39262, which also targets this lesson, at the suggestion of @huyenltnguyen. With this one, I figured since `const xAxis = d3.axisBottom(xScale);` was already on a line by itself, I may as well add block formatting to it.

### Before:

> Here's an example to create the x-axis based on the <code>xScale</code> in the previous challenges:
>
> <code>const xAxis = d3.axisBottom(xScale);</code>

### After:

> Here's an example to create the x-axis based on the <code>xScale</code> in the previous challenges:
>
> ```js
> const xAxis = d3.axisBottom(xScale);
> ```

**Comment by RandellDawson** [comment]
@erictleung Are you OK with the changes?

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by nabraas60000** [comment]
hi

**Review by erictleung — APPROVED** [review]
LGTM! 👍

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**